### PR TITLE
fix: missing answer for javascript question

### LINKED ISSUE #399: [Bug] - Missing answer for JavaScript question
### What bug did you find in the codebase?

It looks like we have a missing answer for this JavaScript question.

Screen Shot 2022-10-15 at 2 50 46 PM

I would also like to change the distractor options. 

Here is the fix I am looking for.
```
  {
    Question: "In JavaScript, what will the following code return: typeof a?",
    Answer: "undefined",
    Distractor1: "string",
    Distractor2: "object",
    Distractor3: "number",
    Explanation:
      "In JavaScript, if you use the typeof operator on an undeclared variable then it will return undefined. ",
    Link: "https://www.freecodecamp.org/news/javascript-typeof-how-to-check-the-type-of-a-variable-or-object-in-js/",
  },
```

This question is located in the JavaScript quiz here:
https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/src/data/javascript-quiz.ts

Please read through the contributing documentation first
https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/c2896c9957c075f3bab7801f7576f9766d8f161d/CONTRIBUTING.md

If you are interested in working on this issue, please reply asking to be assigned.
**Please note: this issue will only be assigned on a first come basis and will only be assigned to those who are new to the Developer Quiz site**

### PR DESCRIPTION
**Pull request by lizwe-mac**
-added a missing answer to Distractor2 which was an empty string before.

 {
    Question: "In JavaScript, what will the following code return: typeof a?",
    Answer: "undefined",
    Distractor1: "string",
    Distractor2: "object",
    Distractor3: "number",
    Explanation:
      "In JavaScript, if you use the typeof operator on an undeclared variable then it will return undefined. ",
    Link: "https://www.freecodecamp.org/news/javascript-typeof-how-to-check-the-type-of-a-variable-or-object-in-js/",
  },

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #399

### CODE REVIEW

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -1249,7 +1249,7 @@ const javascriptQuiz = [
     Question: "In JavaScript, what will the following code return: typeof a?",
     Answer: "undefined",
     Distractor1: "not defined",
-    Distractor2: "",
+    Distractor2: "object",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:1252]: Please make sure to update all of the distractors as outlined in the issue 👍

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/javascript-quiz.ts (hunk 1 above)
Comment (lizwe-mac) [reply src/data/javascript-quiz.ts:1252]: Changes have been implemented

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**